webpack.commons.js

```diff
@@ -29,6 +30,7 @@ module.exports = {
     },
   },
   plugins: [
+    new Dotenv({ systemvars: true }),
```

Comment:
So did you tested to run with env vars from the command line? If `.env` file is defined, which one does Dotenv plugin takes?

Reply:
It will prioritize the system env vars (the one that you provide via CLI). There'll be a demo for that so that you can be certain.

Reply:
Uploaded the demo.